scapy/arch/linux.py

```diff
@@ -216,43 +216,49 @@ def get_alias_address(iface_name, ip_mask, gw_str, metric):
     return
 
 
-def read_routes():
+def read_routes(loopback_name=LOOPBACK_NAME):
```

Comment:
I would really prefer that you let the function use LOOPBACK_NAME directly, and simply mock it while performing the test :/ This stuff will never be used otherwise...

Reply:
ohh - I did expect that kind of reaction :laughing: 

for most programmers it is completely acceptable to inject stuff via global mechanisms (aka creating side effects) - but changing an interface for making things testable (the foundation of creating value while crafting software) is a no go. yes - keep things simple - but clean too. especially code written by other people :smirk: 

but at the end of the day I agree: lets keep things consistent over the whole code base. fixed in fc08b23 & fc08b23.

Reply:
Thanks 😄